controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Action.java

```diff
@@ -18,11 +18,11 @@
     update,
     delete;
 
-    public static EnumSet<Action> all() {
+    static EnumSet<Action> all() {
```

Comment:
Why all the reduction in visibility?

Reply:
Hiding the parts that aren't meant to be used outside the package.

Reply:
The intended API is `Enforcer.allows`, and things to make `Role`s and  `Action`s with.